### ISSUE TITLE
Alias balance issues even with 1.2

### ISSUE THREAD

**Issue by spex6** [syscoin/blockmarket-desktop-public#312]
MCA has been helping me in slack but thought this issue had been resolved for 1.2. I am having these issues with 1.2 installed so I am creating a new issue here. Feel free to merge it with another issue if you feel it is necessary.

I made a purchase using BM 1.2 and had an issue with the transaction showing in my transaction list (see other issue I just logged as it may be related) but I also had an issue with the displayed balance of my alias used to make the purchase. It is still displaying the same balance as before the purchase was made.

I also still have an issue from a transfer I made back when I had version 1.1 installed. I transferred 10 sys to an alias I owned from within the same wallet (this was the first transfer I made to an alias). This transfer appears in my transaction list as -10sys to my alias however the alias still displays a balance of 0. Another slightly odd thing is that the cryptographic address for this alias is also displayed in the my addresses section but this does not happen for my other 2 aliases.

**Comment by spex6** [comment]
Yeah I've restarted BM and the problem persists. I'll retest in 2.2.

**Comment by akriv-git** [comment]
I also had similar problems on BM 1.2:
- alias doesn't show balance when funds are transferred to the alias address from outside BM
- standard address balances on Receiving Addresses are not correct after transfer of an amount large enough that required pulling from several standard addresses.
In both scenarios, Balance on All Accounts is correct.

**Comment by jnessbcf** [comment]
@akriv-git Did this problem go away in BMD3?  If so, let me know and I will close.
